[2021-04-21 20:19:36,203] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-19T20:00:00+00:00 [queued]>
[2021-04-21 20:19:36,250] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-19T20:00:00+00:00 [queued]>
[2021-04-21 20:19:36,251] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:19:36,252] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-21 20:19:36,254] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:19:36,270] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2021-04-19T20:00:00+00:00
[2021-04-21 20:19:36,275] {standard_task_runner.py:52} INFO - Started process 1328 to run task
[2021-04-21 20:19:36,300] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'print_date', '2021-04-19T20:00:00+00:00', '--job-id', '519', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpv_00vr7d', '--error-file', '/tmp/tmphu6g8qm3']
[2021-04-21 20:19:36,303] {standard_task_runner.py:77} INFO - Job 519: Subtask print_date
[2021-04-21 20:19:36,371] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.print_date 2021-04-19T20:00:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:19:36,453] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T20:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T20:00:00+00:00
[2021-04-21 20:19:36,454] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-21 20:19:36,456] {bash.py:158} INFO - Running command: date
[2021-04-21 20:19:36,478] {bash.py:169} INFO - Output:
[2021-04-21 20:19:36,483] {bash.py:173} INFO - Wed Apr 21 20:19:36 UTC 2021
[2021-04-21 20:19:36,484] {bash.py:177} INFO - Command exited with return code 0
[2021-04-21 20:19:36,526] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=print_date, execution_date=20210419T200000, start_date=20210421T201936, end_date=20210421T201936
[2021-04-21 20:19:36,573] {taskinstance.py:1220} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:19:36,588] {local_task_job.py:146} INFO - Task exited with return code 0
